tests/test_parser.py

```diff
@@ -187,3 +188,67 @@ def test_named_color(self):
 
         with self.assertRaises(ValueError):
             parser.color('not a color')
+
+
+class ParseRectTests(TestCase):
+
+    def test_rect_valid(self):
+        # Comma separated
```

Comment:
Insert comment about separating tests here. :-)

Reply:
Split tests (I prefer to move to pytest on a separate PR the fonts one... if that is what you were implying ;-)  )